app/assets/javascripts/components/header/header.es6.jsx

```diff
@@ -81,7 +85,7 @@ class Header extends Component {
     if (this.props.active) {
       return (
         <Clickable
-          action={null}
+          action={() => {ProfileActions.toggleSidebar(this.state.profile.id, !this.state.profile.has_sidebar);}}
```

Comment:
We can just pull this out into a helper method `toggleSidebar` since it's a bit long. :+1: